devtools/conda-recipe/meta.yaml

```diff
@@ -40,7 +40,7 @@ requirements:
     - openmm
     - parmed
     - openmoltools
-    - alchemy
+    - alchemy >=1.1.3
```

Comment:
Nothing wrong here, but does the current conda version of alchemy have the `softcore_beta` parameter? I had some issue with that, and had to install directly from the git repo. Is there no `alchemy-dev`? I couldn't get that either.